### DISCUSSION TITLE
Extract vocals without reverb ?

### DISCUSSION THREAD

**Discussion by xGhostOfSparta** [Anjok07/ultimatevocalremovergui#747] [Q&A] [answered]
hey,
 is it possible to extract vocal without the reverb ?
 thanks

**Comment by deadniell** [comment]
I just created an account to follow this request. Someone pls help

**Comment by ghost** [comment, marked as answer]
There is a model available for UVR5 that can do de-echo and de-reverb. [https://imgur.com/YF44YYP](https://imgur.com/YF44YYP)
What I do is simply split into instrumentals and voice and use the dereverb model afterwards. Might also work if you first use dereverb and split afterwards, quality depends on the source track.

**Comment by ghost** [comment]
> is it the UVR5 that adding the reverb ot its just the original reverb from the song ?

Its 99% original Reverb. I have never seen any UVR included models create
Reverb where there wasnt one in the input file, in contrast to legacy vocal
removal and noise reduction methods, which could introduce
reverb/delay-like artifacts quite often

(I hope my reply doesnt look out of place, for some reason your message
doesnt show up on github)

DANIEL120321 ***@***.***> schrieb am Fr., 1. Sep. 2023, 18:10:
